heft-plugins/heft-api-extractor-plugin/src/ApiExtractorPlugin.ts

```diff
@@ -122,7 +122,11 @@ export default class ApiExtractorPlugin implements IHeftTaskPlugin {
       heftConfiguration
     );
     const apiExtractorConfigurationObject: TApiExtractor.IConfigFile =
-      apiExtractorPackage.ExtractorConfig.loadFile(apiExtractorConfigurationFilePath);
+      apiExtractorPackage.ExtractorConfig.loadFile(apiExtractorConfigurationFilePath, {
+        apiReport: {
+          reportTempFolder: taskSession.tempFolderPath
```

Comment:
Couldn't we simply set `reportTempFolder` in `api-extractor.json` in the rig?

(Generally we aim for the config file is meant to represent the tool's configuration, not an input to a mysterious black box that produces the actual configuration.)